pkg/pillar/worker/worker.go

```diff
@@ -20,7 +20,27 @@ type Logger interface {
 }
 
 // Worker captures the worker channels
```

Comment:
Comment is out of date I think, since this defines the interface now.

Reply:
Nice. Fixing.

Reply:
Done